src/pages/Overview/BalanceLinks.tsx

```diff
@@ -22,28 +24,15 @@ export const BalanceLinks = () => {
           lg
           onClick={() =>
             window.open(
-              `https://${name}.subscan.io/account/${activeAccount}`,
+              `https://${networkNameDomainPart}.subscan.io/account/${activeAccount}`,
               '_blank'
             )
           }
           iconRight={faExternalLinkAlt}
           iconTransform="shrink-2"
           text="Subscan"
           marginRight
-          disabled={!activeAccount}
-        />
-        <ButtonPrimaryInvert
-          lg
-          onClick={() =>
-            window.open(
-              `https://${name}.polkawatch.app/nomination/${activeAccount}`,
-              '_blank'
-            )
-          }
-          iconRight={faExternalLinkAlt}
-          iconTransform="shrink-2"
-          text="Polkawatch"
-          disabled={!(activeAccount && ['polkadot', 'kusama'].includes(name))}
+          disabled={name !== NetworkNameEnum.AlephZero}
```

Comment:
Not supporting testnet / devnet?

Reply:
no, I don't think we have subscan for either of them